### PR TITLE
add tags and a README

### PR DESCRIPTION
**Pull request by duran-thomas**

### CODE REVIEW

File: tests/account_test.js [hunk 1]
```diff
@@ -11,7 +11,9 @@ Scenario("I can verify that I am on the correct obit", ({ I, obitPage }) => {
   );
   // Then I should see her name in the heading
   I.see("Virginia", obitPage.nameHeader);
-});
+})
+  .tag("@JanusBeta")
+  .tag("@prodSafe");
```
Comment (pegfoltz) [inline tests/account_test.js:17]: Not an issue for this project, but our repo uses @productionSafe, just FYI (and our tools enforce it if the environment we're pointing to is production)

File: tests/account_test.js [hunk 2]
```diff
@@ -27,9 +29,11 @@ Scenario("I can add an entry to the guestbook", ({ I }) => {
     "Duran",
     "dthomas@qualityworkscg.com"
   );
-});
+})
+  .tag("@JanusBeta")
+  .tag("@prodSafe");
```
Comment (pegfoltz) [inline tests/account_test.js:34]: Typically tags are in camelCase

**Review by pegfoltz — APPROVED** [review]
A couple of minor mentions about tagging, looks good but let Gilbert look it over before merging.

**Review by ghopkins-legacy — APPROVED** [review]
aside from what peg mentioned. 

LGTM!

**Pull request merged**